utils/tests/createImaAppTests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Add customized environment configuration
+mv server/config/environment.js server/config/environment.orig.js
+cp "$ROOT_DIR_IMA/utils/tests/app/environment.js" server/config/environment.js
+NODE_ENV=prod node server/server.js &
+IMA_SKELETON_SERVER_PID=$!
+
+sleep 7
+
+echo "Run benchmark test on create-ima-app"
+cd "$ROOT_DIR_IMA"
+node_modules/.bin/autocannon -c $PARALLEL_TEST_CONNECTIONS --no-progress "$TARGET_WEB_URL"
+
+
+cd "$ROOT_DIR_IMA_APP"
+
+echo "Run lint for create-ima-app"
+npm run lint
+
+echo "Test unit and integration tests for create-ima-app"
+npm run test
+
+kill $IMA_SKELETON_SERVER_PID
```

Comment:
There is no need for server to be running for `npm run lint`, or `npm run test`. So I feel like its better to just kill this process right after benchmark test.

Reply:
Newly `kill` command is set right after benchmark test.